src/core/ColorPicker/ColorPicker.test.tsx

```diff
@@ -506,10 +526,27 @@ it('should handle hsl input change', () => {
   const h = container.querySelectorAll('input')[0] as HTMLInputElement;
   const s = container.querySelectorAll('input')[1] as HTMLInputElement;
   const l = container.querySelectorAll('input')[2] as HTMLInputElement;
+  const a = container.querySelectorAll('input')[3] as HTMLInputElement;
   expect(h).toBeTruthy();
   expect(s).toBeTruthy();
   expect(l).toBeTruthy();
+  expect(a).toBeTruthy();
+
+  // Should not update if value hasn't changed
+  fireEvent.change(h, { target: { value: '0' } });
+  fireEvent.keyDown(h, { key: 'Enter' });
+
+  fireEvent.change(s, { target: { value: '0' } });
+  fireEvent.keyDown(s, { key: 'Enter' });
+
+  fireEvent.change(l, { target: { value: '0' } });
+  fireEvent.keyDown(l, { key: 'Enter' });
+
+  fireEvent.change(a, { target: { value: '1' } });
+  fireEvent.keyDown(a, { key: 'Enter' });
+  expect(handleOnChange).not.toHaveBeenCalled;
```

Comment:
This line is doing nothing because you are not calling the method

```suggestion
expect(handleOnChange).not.toHaveBeenCalled();
```

Reply:
Fixed